### DISCUSSION TITLE
nwg-shell-config 0.5.46 (2024-08-21)

### DISCUSSION THREAD

**Discussion by nwg-piotr** [nwg-piotr/nwg-shell#381] [Changelog]
https://github.com/nwg-piotr/nwg-shell-config/releases/tag/v0.5.46

@Antiz96: sorry, but it was a pretty serious bug and had existed for a long time.

**Comment by Antiz96** [comment]
No problem, v0.5.46 released in [extra] ;)

Just so you know, I have my own alerting mechanism for new upstream releases so you don't *have to* tag me on new releases (but feel free to do so if you think it's urgent/relevant though).
I'm usually fairly fast at packaging/releasing new upstream releases.

**Reply by nwg-piotr** [reply]
Thank you!

> Just so you know, I have my own alerting mechanism

Very well, but it's always nice to talk to you. :)

**Reply by Antiz96** [reply]
Heyhey ;D
